codespell_lib/data/dictionary.txt

```diff
@@ -44818,6 +44819,7 @@ protcools->protocols
 protcted->protected
 protecion->protection
 protecte->protected, protect,
+protecten->protection, protected,
```

Comment:
I think this is protect enable. @matkoniecz may remember the context from #2146 ? Or @sebweb3r from https://github.com/codespell-project/codespell/pull/1721/commits/806ea830581957823a19327f3c76c345a8603da4

Reply:
There's an occurrence in [Linux](https://github.com/torvalds/linux/blob/7c16f0a4ed1ce7b0dd1c01fc012e5bde89fe7748/drivers/staging/vt6655/mac.h#L267):
```c
#define ENCFG_CFP_PROTECTEN 0x00000040
```
I'm not certain it's enough to warrant an exception, but let's keep it for now.